specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/webapplicationfirewall.json

```diff
@@ -295,6 +295,15 @@
             "$ref": "#/definitions/WebApplicationFirewallCustomRule"
           }
         },
+        "managedRules": {
+          "description": "Describes managed rules inside the policy.",
+          "$ref": "#/definitions/ManagedRuleSets"
+        },
+        "exclusions":{
+          "description": "Describes the Exclusions that are applied on the policy.",
+          "$ref": "#/defintions/CrsExclusions"
```

Comment:
Typo in `defintions`. Also, you don't really need to move this to definitions. This code would be fine:
```json
"exclusions": {
  "type": "array",
  "items": {
    "$ref": "#/definitions/CrsExclusionEntry"
  },
  "description": "Describes the Exclusions that are applied on the policy."
}
```

Reply:
Addressed!